scene/2d/sprite.cpp

```diff
@@ -339,6 +339,8 @@ bool Sprite::is_pixel_opaque(const Point2 &p_point) const {
 			mirror_x = (int)(q.x / texture->get_size().width);
 			mirror_y = (int)(q.y / texture->get_size().height);
 		}
+		if (texture->get_size().width == 0 || texture->get_size().height == 0)
```

Comment:
You should likely move the code before the previous block, otherwise you already did a division by zero.

Reply:
Good catch, thank you! I forgot to account for if the texture is also mirror repeating. I am changing it now.